From c6f0b9f195416e2d3c474e51190b6aa0c238aa36 Mon Sep 17 00:00:00 2001
From: Guoqing Jiang <gqjiang@suse.com>
Date: Mon, 14 Mar 2016 17:01:38 +0800
Subject: [PATCH] md/bitmap: remove redundant return in bitmap_checkpage
Git-commit: c6f0b9f195416e2d3c474e51190b6aa0c238aa36
Patch-mainline: v4.6-rc1
References: fate#320291

The "return 0" is not needed since bitmap_checkpage
will finally return 0 for the case.

Signed-off-by: Guoqing Jiang <gqjiang@suse.com>
Signed-off-by: Shaohua Li <shli@fb.com>
Signed-off-by: Coly Li <colyli@suse.de>

---
 drivers/md/bitmap.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/md/bitmap.c b/drivers/md/bitmap.c
index bbe7b64..7df6b4f 100644
--- a/drivers/md/bitmap.c
+++ b/drivers/md/bitmap.c
@@ -98,7 +98,6 @@ __acquires(bitmap->lock)
 		   bitmap->bp[page].hijacked) {
 		/* somebody beat us to getting the page */
 		kfree(mappage);
-		return 0;
 	} else {
 
 		/* no page was in place and we have one, so install it */
-- 
2.6.6

